packages/resolve-scripts/src/server/auth/createAuthOptions.js

```diff
@@ -0,0 +1,27 @@
+export default (authReq, authRes, next) => ({
+  onSuccess: (options, user, info) => {
+    authRes.applyJwtValue(user, authRes.expressRes, options.successRedirect)
+  },
+  onFail: (options, challenge, status) => {
+    if (options.failureRedirect) {
+      const res = authRes.expressRes
+      res.statusCode = 302
+      res.setHeader('Location', options.failureRedirect)
+      res.setHeader('Content-Length', '0')
+      res.end()
+    }
+  },
+  onRedirect: (options, url, status) => {
+    const res = authRes.expressRes
+    res.statusCode = status || 302
+    res.setHeader('Location', url)
+    res.setHeader('Content-Length', '0')
+    res.end()
```

Comment:
Looks like copy-paste from onFail method, usually such code take out to separate function

Reply:
Is this really important?

Reply:
It's not critical, just for learning